pkg/controller/clusterinstallation/mattermost.go

```diff
@@ -34,10 +34,21 @@ func (r *ReconcileClusterInstallation) checkMattermostIngress(mattermost *matter
 }
 
 func (r *ReconcileClusterInstallation) checkMattermostDeployment(mattermost *mattermostv1alpha1.ClusterInstallation, reqLogger logr.Logger) error {
-	dbPassword, err := r.getMySQLSecrets(mattermost, reqLogger)
-	if err != nil {
-		return errors.Wrap(err, "Error getting the database password.")
+	externalDB := false
+	var dbPassword string
```

Comment:
It's a little unclear to me, but I just want to check that we don't want to set a username of password here if an external DB is used. I think that makes sense based on my understanding of what this code is doing, but I did want to check as we are setting the values if an external DB is not used.

Reply:
if we use an external DB we should set the DSN in the secret like `mysql:/USER:PASS@tcp(HOST:3306)/mattermost?charset=utf8mb4,utf8&readTimeout=30s&writeTimeout=30s`